src/tile.cpp

```diff
@@ -539,7 +539,7 @@ ReturnValue Tile::queryAdd(int32_t, const Thing& thing, uint32_t, uint32_t flags
 				}
 			}
 
-			MagicField* field = getFieldItem();
+			auto field = getFieldItem();
```

Comment:
Could this be `const auto`
```suggestion
			const auto field = getFieldItem();
```